lua/lspconfig/erlangls.lua

```diff
@@ -0,0 +1,31 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+
+configs.erlangls = {
+  default_config = {
+    cmd = { "erlang_ls" };
+    filetypes = {"erlang"};
+    root_dir = function(fname)
+      return util.root_pattern("rebar.config", "erlang.mk", ".git")(fname) or vim.loop.os_homedir()
+    end;
+    settings = {};
+  };
+  docs = {
+    description = [[
+https://erlang-ls.github.io
+
+Language server for the Erlang.
+
+```lua
```

Comment:
You don't need this since it's on your path

Reply:
What do you mean?